src/lib.rs

```diff
@@ -230,3 +250,44 @@ impl Wigner9j {
         }
     }
 }
+
+/// symmetrized version of second type of Wigner 12-j symbol
```

Comment:
Is there a reference that can be cited regarding the 12j symbol regarding the convention and formula adopted here?

The 12j symbol is a bit more esoteric than the others, so I worry the definition and phase convention might not be as standardized in literature.

Reply:
My implemantation is based on the equation (19.3) on the following books (P.62).

A. P. Yutsis, I. B. Levinson, and V. V. Vanagas,Mathmatical Apparatus of the Theory of Angular Momentum(Israel Program for Scientific Translations, Jerusalem,1962).

Reply:
Thanks. Would you mind including this reference in the documentation?